assignments/arrays.js

```diff
@@ -63,34 +63,72 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+
+     const getID = inventory.find(function(elem) {
+         return elem.id === 33;
+     }
+     )
+
+console.log (`Car 33 is a ${getID.car_year} ${getID.car_make} ${getID.car_model}`);
+// console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
 
 
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
-
+let lastCar = inventory.find(function(elem) {
```

Comment:
I can understand you wanted to use the `find` method and it is really great that you figured out how to use it. But here `lastCar = inventory[inventory.length - 1]` would have ben more appropriate.